updater/manager/AddonManager.py

```diff
@@ -9,6 +12,11 @@
 
 from updater.site import SiteHandler
 
+def format_path(path: str):
```

Comment:
I'm trying to creep towards type hinting on everything, can you add `-> str` return type hint"?

Reply:
I am fairly inexperienced with Python 3, so this is new to me.
I'll make sure the hint is present on the next PR.

Reply:
If you commit it to your branch `fspy:wsl`, then it will be added to this PR